### PR TITLE
update libg for localization updates

### PR DESCRIPTION
**Pull request by mjkkirschner**
### Purpose

* Obsolete node messages in LibG are now localized.
* B-spline approximation fixes.

### Declarations

Check these if you believe they are true

- [ ] The codebase is in a better state after this PR
- [ ] Is documented according to the [standards](https://github.com/DynamoDS/Dynamo/wiki/Coding-Standards)
- [ ] The level of testing this PR includes is appropriate
- [ ] User facing strings, if any, are extracted into `*.resx` files
- [ ] All tests pass using the self-service CI.
- [ ] Snapshot of UI changes, if any.
- [ ] Changes to the API follow [Semantic Versioning](https://github.com/DynamoDS/Dynamo/wiki/Dynamo-Versions) and are documented in the [API Changes](https://github.com/DynamoDS/Dynamo/wiki/API-Changes) document.
- [ ] This PR modifies some build requirements and the readme is updated

### Release Notes

(FILL ME IN) Brief description of the fix / enhancement. **Mandatory section** 

### Reviewers

(FILL ME IN) Reviewer 1  (If possible, assign the Reviewer for the PR)

(FILL ME IN, optional) Any additional notes to reviewers or testers.

### FYIs

(FILL ME IN, Optional) Names of anyone else you wish to be notified of

### CODE REVIEW

File: src/DynamoCore/DynamoCore.csproj [hunk 1]
```diff
@@ -23,8 +23,8 @@
   <ItemGroup>
     <PackageReference Include="Autodesk.IDSDK" Version="1.1.6" />
     <PackageReference Include="DynamoVisualProgramming.Analytics" Version="3.0.0.2506" />
-    <PackageReference Include="DynamoVisualProgramming.LibG_229_0_0" Version="2.16.0.3176" GeneratePathProperty="true" />
```
Comment (QilongTang) [inline src/DynamoCore/DynamoCore.csproj:26]: Are supposed to bump up to 2.17.0?

Comment (mjkkirschner) [reply src/DynamoCore/DynamoCore.csproj:26]: not super important, I can't remember what the versioning strategy is for LibG at the moment.

Comment (mjkkirschner) [reply src/DynamoCore/DynamoCore.csproj:26]: in a separate PR maybe it should just get bumped all the way to 2.18

**Comment by aparajit-pratap** [comment]
Just pointing out here that this drop will also include the recent B-spline approximation fixes.

**Review by pinzart90 — APPROVED** [review]

**Pull request closed without merging**